dialect/sql/schema/sqlite.go

```diff
@@ -119,6 +123,52 @@ func (*SQLite) cType(c *Column) (t string) {
 	return t
 }
 
+func (d *SQLite) typeField(c *Column, str string) error {
+	switch parts := typeFields(str); parts[0] {
+	case "int", "integer":
+		c.Type = field.TypeInt32
+	case "smallint":
+		c.Type = field.TypeInt16
+	case "bigint":
+		c.Type = field.TypeInt64
+	case "tinyint":
+		c.Type = field.TypeInt8
+	case "double", "real":
+		c.Type = field.TypeFloat64
+	case "timestamp", "datetime":
```

Comment:
> "timestamp"

What is usually used by other ORMs? I know that `date` and `datetime` types exist in `sqlite`, but not familiar with `timestamp`.

Reply:
timestamp is used in postgres at least -- it's SQL 92 IIRC. There is also `timestamp with time zone`. the NOW() function typically returns a timestamp of some fashion in SQL.

I need to do a second pass and make sure all the types line up, I think I took this from the mysql driver. Sorry if reviewing this caused you a headache.